libs/gl-cli/src/util.rs

```diff
@@ -0,0 +1,91 @@
+use bip39::{self, Language, Mnemonic};
+use dirs;
+use gl_client::bitcoin::secp256k1::rand::{self, RngCore};
+use gl_client::credentials;
+use std::path::PathBuf;
+use std::{
+    fs::{self, File},
+    io::Write,
+    path::Path,
+};
+use thiserror;
+
+pub const SEED_FILE_NAME: &str = "hsm_secret";
+pub const CREDENTIALS_FILE_NAME: &str = "credentials.gfs";
+pub const DEFAULT_GREENLIGHT_DIR: &str = "greenlight";
+
+// -- Seed section
+
+pub fn generate_seed() -> Result<Vec<u8>> {
+    let mut entropy = [0u8; 32];
+    let mut rng = rand::thread_rng();
```

Comment:
I had to quickly check whether the rng returned by this is a CSPRNG, and it seems it is: https://rust-random.github.io/rand/rand/rngs/struct.ThreadRng.html :+1: